### PR TITLE
Fixes #6432: We need a command to show rudder detailed informations

### PR DESCRIPTION
**Pull request by peckpeck**
http://www.rudder-project.org/redmine/issues/6432

### CODE REVIEW

File: share/commands/agent-info [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  ROLES=", rudder-agent"
+fi
+
+if [ -e /opt/rudder/etc/disable-agent ]; then
+  ENABLED="disabled"
+else
+  ENABLED="enabled"
+fi
+
+echo "UUID: ${UUID}"
```
Comment (jooooooon) [inline share/commands/agent-info]: I would love to see the hostname here too!

**Comment by jooooooon** [comment]
Apart from that this looks great :)

**Comment by peckpeck** [comment]
PR updated

**Comment by Kegeruneku** [comment]
Cool! Merging :dancer:

**Pull request merged**